src/graphql/models/UserAwardedBadge.js

```diff
@@ -0,0 +1,17 @@
+import {
+  GraphQLBoolean,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+} from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'UserAwardedBadge',
+  fields: () => ({
+    badgeId: { type: new GraphQLNonNull(GraphQLString) },
+    badgeMetaData: { type: new GraphQLNonNull(GraphQLString) },
+    isDisplayed: { type: new GraphQLNonNull(GraphQLBoolean) },
+    createdAt: { type: new GraphQLNonNull(GraphQLString) },
+    updatedAt: { type: new GraphQLNonNull(GraphQLString) },
```

Comment:
nit: when we implement the badge dropdown selector in user profile page, it will be useful to add a field that resolves to `Badge` here, so that we can get the data needed to render the dropdown via:

```graphql
{
  GetUser {
    badges {
      isDisplayed
      badge {
        name
      }
    }
  }
}
```

That could be added later, along with the mutation that enables users to switch their major badge.